Sources/RxMoya/RxMoyaProvider.swift

```diff
@@ -21,7 +21,10 @@ open class RxMoyaProvider<Target>: MoyaProvider<Target> where Target: TargetType
 
         // Creates an observable that starts a request each time it's subscribed to.
         return Observable.create { [weak self] observer in
```

Comment:
I can't recall, could someone check in on that and get back to us? Really swamped right now 😬